bin/tpaexec

```diff
@@ -1091,10 +1101,33 @@ esac
 # If we can find a venv located relative to TPA_DIR, we activate it as a
 # convenience.
 
+if [ -z ${PYTHON+x} ]; then
+    PYTHON=/usr/libexec/edb-python39/bin/python3
+    if [ ! -f $PYTHON ]; then
+        PYTHON=python3
+    else
+        echo "Running edb-python $PYTHON"
```

Comment:
Leftover debugging noise?

Reply:
No, I thought it was a good idea to have a line like this if we received any bug report.

Reply:
It's not a good idea to add this output to *every* tpaexec invocation. `tpaexec info` will show the interpreter, that's enough for our purposes.

Reply:
Done